{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: oops", "trace": "def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\ntest_allure.py:24: Exception"}, "start": 1617541952085, "stop": 1617541952085, "uuid": "ab28a899-47cf-4c37-a608-ce4ad6f9f142", "historyId": "985c3d972d231f87d0914b89f78754fd", "testCaseId": "cb41bd139e929d9eef92825c451cc27a", "fullName": "test_allure#test_broken", "labels": [{"name": "suite", "value": "test_allure"}, {"name": "host", "value": "LAPTOP-UL20B91J"}, {"name": "thread", "value": "21336-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_allure"}]}